.chloggen/enable-goleak-oidcauthextension.yaml

```
@@ -0,0 +1,30 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: oidcauthextension
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Enable goleak in oidcauthextension and fix a goleak issue while shutdown.
```

```suggestion
note: Fix a goroutine leak during shutdown.
```
Nit: Enabling `goleak` is for testing, so not really relevant to the end user. Also, the component name will be included from `component` with the changelog message, so it's not necessary to include in this message as well.